init.lua

```diff
@@ -398,7 +398,18 @@ function lib_mount.drive(entity, dtime, is_mob, moving_anim, stand_anim, jump_he
 					pass:set_hp(pass:get_hp() - intensity)
 				end
 				local pos = entity.object:get_pos()
-				minetest.add_item(pos, entity.drop_on_destroy)
+
+				-- Handle drops
+				local i = math.random(1, #entity.drop_on_destroy)
+				local j = math.random(2, #entity.drop_on_destroy)
```

Comment:
This needs an explanation as to what the chance here calculated is. Previously there was 100% chance, is this compatible?

Reply:
`entity.drop_on_destory` is table which stores all the items that will be dropped on destroy.
It will drop one of those items, from `1` to the length of the table.

See https://github.com/Panquesito7/vehicle_mash/blob/master/init.lua#L90.

Reply:
The explanation should be in the code, a readme, or an API document, not in a github comment :)

Reply:
Where's the documentation?